DANS-KNAW/easy-deposit
# Rename deposit to sword2

~~fixes EASY-~~
#### When applied it will
- rename the `Easy-Deposit` project to `Easy-Sword2`
#### Where should the reviewer @DANS-KNAW/easy start?
- Please do not only focus on the changes I made, but also to the rest of the code/text/...! Did I miss any occurrences of `deposit` that need to be changed to `sword2`? Not that this is not trivial, as `deposit` was used kind of ambiguously in this project: it can refer to a 'real' deposit or refer to the project's name. Only the latter should be changed to `sword2`!
- **IF YOU HAVE A FORK OF THIS PROJECT, I KINDLY REQUEST YOU CHANGE THE NAME OF THIS FORK ACCORDINGLY!**
- **PLEASE CHANGE THE REMOTE(S) ON YOUR LOCAL CLONES TO POINT TO THE NEW NAME/URL.** [See here for instructions.](https://help.github.com/articles/changing-a-remote-s-url/)
- **PLEASE CHANGE THE NAME OF THE FOLDER OF YOUR LOCAL CLONE ACCORDINGLY.**
#### How should this be manually tested?

This should be tested together with other PRs yet to come in Easy-DTAP and Easy-Build. The other related PR's just changes some textual references and do not influence the code. To test this:
- check out these PRs
- do a `mvn clean install`
- `vagrant destroy && vagrant up` should (after lots of waiting) create a new instance of `deasy`
- _optionally:_ you can try the procedures as described in `README.md` to ensure that everything is still working according to plan.
#### related pull requests on github

| repo | PR | note |
| --- | --- | --- |
| easy-build | [PR#7](https://github.com/DANS-KNAW/easy-build/pull/7) | small change in `pom.xml` |
| easy-dtap | [PR#19](https://github.com/DANS-KNAW/easy-dtap/pull/19) |  |
| easy-license-creator | [PR#6](https://github.com/DANS-KNAW/easy-license-creator/pull/6) | small change in `Functional requirements.md` |
| easy-stage-dataset | [PR#87](https://github.com/DANS-KNAW/easy-stage-dataset/pull/87) | small change in `README.md` |
| easy-ingest-flow | [PR#47](https://github.com/DANS-KNAW/easy-ingest-flow/pull/47) | small changes in `pom.xml` and `README.md` |
| easy-split-multi-deposit | [PR#30](https://github.com/DANS-KNAW/easy-split-multi-deposit/pull/30) | small changes in `README.md` and `CommandLineOptions.scala` |
